src/main/java/seedu/address/model/company/Company.java

```diff
@@ -47,6 +76,8 @@ public Company(CompanyName companyName, CompanyPhone phone,
         this.companyDescription = companyDescription;
         this.tags.addAll(tags);
         this.internships = new UniqueInternshipList();
+        this.filteredInternshipsRaw = new FilteredList<>(this.internships.asUnmodifiableObservableList());
+        this.filteredInternships = new SortedList<>(this.filteredInternshipsRaw, internshipComparator);
```

Comment:
Unfortunately due to the feature freeze, I don't think we're allowed to change the UI output :(